org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/DefaultGradleBuild.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2018 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.buildship.core.internal;
+
+import org.gradle.tooling.CancellationTokenSource;
+import org.gradle.tooling.GradleConnector;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Preconditions;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+
+import org.eclipse.buildship.core.GradleBuild;
+import org.eclipse.buildship.core.SynchronizationResult;
+import org.eclipse.buildship.core.internal.workspace.NewProjectHandler;
+
+/**
+ * Default implementation for {@link GradleBuild}.
```

Comment:
Obvious comment is obvious :)

Reply:
This is a long-standing request of yours. We have a checkstyle rule in place that enforces to have minimal comment on the classes.  I guess it's the right time to get rid of it.